packages/odyssey-react/src/components/TextArea/TextArea.module.scss

```
@@ -78,3 +78,19 @@
     color: var(--PlaceholderTextColor);
   }
 }
+
+.noneResize {
+  resize: none;
+}
+
+.horizontalResize {
+  font-size: horizontal;
+}
+
+.verticalResize {
+  font-size: vertical;
+}
+
+.bothResize {
+  font-size: both;
```

```suggestion
  resize: both;
```

same goes for the other utility classes above.